packages/server/docs/openapi.yaml

```
@@ -0,0 +1,1933 @@
+      in: query
+      description: Whether to include deleted
+      required: false
+      schema:
+        type: boolean
+    latitudeQuery:
+      name: latitude
+      in: query
+      description: Latitude, probably for current user
+      required: true
+      schema:
+        type: number
+        example: 40.015
+    longitudeQuery:
+      name: longitude
+      in: query
+      description: Longitude, probably for current user
+      required: true
+      schema:
+        type: number
+        example: -105.2705
+    slackTimestampHeader:
+      in: header
+      name: X-Slack-Request-Timestamp
+      required: true
+      schema:
+        type: string
+        format: date-time
+    subcategoryPath:
+      name: subcategory
+      in: path
+      description: Single subcategory stub name.
+      example: food_pantry
+      required: true
+      schema:
+        type: string
+
+  responses:
+    EmptyObject:
+      description: Empty json object.
+      content:
+        application/json:
+          schema:
+            type: object
+    ErrorAlreadyExists:
+      description: Resource already exists and cannot be created again.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorObj'
+    ErrorAuthentication:
+      description: Invalid user credentials submitted.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorObj'
+    ErrorInvalidRequest:
+      description: Known invalid request.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorObj'
+    ErrorNotFound:
+      description: Resource invalid or not found.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorObj'
+    ErrorUnknown:
+      description: Unknown invalid request.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorObj'
+    SlackBotResponse:
+      description: Slack Response.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/SlackBotResponse'
+
+  requestBodies:
+    EmptyRequest:
+      description: Empty request body
+      content:
+        application/json: {}
+    SlackBotRequest:
+      description: 'Request body from slack / command.'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/BotSlackJobPost'
+        application/x-www-form-urlencoded:
+          schema:
+            $ref: '#/components/schemas/BotSlackJobPost'
+
+tags:
+  - name: alert
+    description: 'Create and get alerts for display.'
+  - name: bot
+    description: 'Endpoints for slack bot to interact with.'
+  - name: category
+    description: 'Categories'
+  - name: sub-category
+    description: 'SubCategories'
+  - name: eventlogs
+    description: 'Event logs'
+  - name: hotlines
+    description: 'Hotlines'
```

```suggestion
  - name: hotlines
    description: 'Hotlines'
  - name: issues
    description: 'Issues reported on resources'
```
I could see issues being their own tag and tagging all the `resources/issue`(s) with it.